From 3d285e7dd913b406de332bb0b20644f1d4217eb9 Mon Sep 17 00:00:00 2001
From: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
Date: Wed, 14 Aug 2024 06:39:47 +0000
Subject: [PATCH] Fix to boot A15 with Aidl Gatekeeper

Falling back to the orginal Sepolicy Object

Tests Done:
1. Flash the image Bare metal or CiV
2. Image is booting and service is running

Tracked-On: OAM-123432

Signed-off-by: Aman Bhadouria <aman.bhadouria@intel.com>
---
 gatekeeper/aidl/software/file_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gatekeeper/aidl/software/file_contexts b/gatekeeper/aidl/software/file_contexts
index 23a62ea9df..cc4180e33e 100644
--- a/gatekeeper/aidl/software/file_contexts
+++ b/gatekeeper/aidl/software/file_contexts
@@ -1,3 +1,3 @@
 (/.*)?                                                          u:object_r:vendor_file:s0
 /etc(/.*)?                                                      u:object_r:vendor_configs_file:s0
-/bin/hw/android\.hardware\.gatekeeper-service\.nonsecure        u:object_r:hal_gatekeeper_remote_exec:s0
+/bin/hw/android\.hardware\.gatekeeper-service\.nonsecure        u:object_r:hal_gatekeeper_default_exec:s0
-- 
2.34.1

